js/form.js

```diff
@@ -0,0 +1,22 @@
+// Модуль отвечающий за перевод страницы в неактивное состояние
+
+const adForm = document.querySelector('.ad-form');
+const fieldsetBlocks = adForm.querySelectorAll('fieldset');
+
+// Блокируем содержимое блока
+const disableAdForm = () => {
+  adForm.classList.add('ad-form--disabled');
+  fieldsetBlocks.forEach((fieldsetBlock) => {
+    fieldsetBlock.setAttribute('disabled', 'disabled');
```

Comment:
```suggestion
    fieldsetBlock.disabled = true;
```
лучше использовать такой подход